core/iwasm/libraries/debug-engine/handler.c

```diff
@@ -383,7 +383,7 @@ send_thread_stop_status(WASMGDBServer *server, uint32 status, korp_tid tid)
 
     if (status == 0) {
         os_mutex_lock(&tmpbuf_lock);
-        snprintf(tmpbuf, MAX_PACKET_SIZE, "W%02" PRIx32, status);
+        (void)snprintf(tmpbuf, MAX_PACKET_SIZE, "W%02" PRIx32, status);
```

Comment:
why do you sometimes ignore snprintf failure and sometimes check it?

Reply:
If the return value is not used by the following `snprintf()`.